src/Engine/Entity/Entity.hpp

```diff
@@ -123,6 +123,12 @@ class Entity {
          * @return The model matrix.
          */
         glm::mat4 GetModelMatrix() const;
+
+        /// Get the local model matrix.
+        /**
+        * @return The local model matrix.
+        */
```

Comment:
Increment documention with one space. Example:
/// Comment
/**
 *  @return The local...
 */